operator_ui/src/components/Jobs/RegionalNav.js

```diff
@@ -79,7 +79,7 @@ const RegionalNav = ({
   const definition = job && jobSpecDefinition(job)
   const handleClick = () => {
     createJobRun(job.id, SuccessNotification, ErrorMessage).then(() =>
-      fetchJob(job.id)
+      fetchJobRuns(job.id, 1, 5)
```

Comment:
@NavyAdmiral how do you feel about extracting these magic numbers into `const`'s? e.g. `DEFAULT_PAGE`, `RECENT_RUNS_COUNT`